lib/elixir/lib/code.ex

```diff
@@ -650,6 +650,10 @@ defmodule Code do
       when non-existing atoms are found by the tokenizer.
       Defaults to `false`.
 
+    * `:warn_on_unnecessary_quotes` - when `false`, does not warn
+      when atoms, keywords or calls have unnecessary quotes on
+      them.
```

Comment:
I think the documentation about the default value is handy :)

I just want to be sure that part was not overlooked from my previous comments. If you disagree, no worries :)

Reply:
I missed it. :) Tks for double checking.